### DISCUSSION TITLE
Layout of sources

### DISCUSSION THREAD

**Discussion by phillipgardiner** [open-sdg/open-sdg#1551] [Ideas]
@AnnCorp @brockfanning @LucyGwilliamAdmin @otis-bath  During the tech call 25th January 2022, a bug was raised (https://github.com/open-sdg/open-sdg/issues/1532) in regard to the layout of the sources associated with an indicator (on the UK site). This bug raised the possibility of changing the layout of the sources listed for an indicator. This ticket kicks off a discussion on this topic.

A little bit of background digging has been done in order to help guide any decision around this.

A number of suggestions were propsed including:
- Single column (stack each source under each other)
- Each source to have its own tab
- Place sources in some type of accordian 

Ultimately, if sources are to remain in thier current tabs it is not receommended to have any additional component (e.g. accordian) within the tab. Additionally, not all users will notice accordians or understand how they work. 

For this reason its is believed there are two main choices availiable:
1) Single column (stack each source under each other)
2) Each source has its own tab

One potential limitation of a single column approach is that depending on the number of sources for an indicator the page could end up quite long and require a large amount of scrolling.

One potential limitaiton of a tab apporach is comparing information in different tabs is difficult - having to memorise any information and switch backwards and forwards can be frustrating and difficult. It is currently not known if the need to compare across sources is a user need around sources?

In order to provide some data / guidence behind these options I have pulled together a table which shows for the UK, on average how many data sources an indicator may have (see attachment). As can be seen from the table approximately 70% of indicators have either 0 or 1 data sources, and 30% have 2 or more (12% have 3 or more). 
[Number of sources.xlsx](https://github.com/open-sdg/open-sdg/files/7941130/Number.of.sources.xlsx)

As a side note we must also remeber we are looking at implemtnting a new design to tabs when displayed on mobile (https://github.com/open-sdg/open-sdg/issues/1054). This effectively would change the mobile view to a scrolling type approach.

@LucyGwilliamAdmin @AnnCorp @brockfanning @otis-bath  - to be discussed

**Comment by LucyGwilliamAdmin** [comment]
@phillipgardiner @otis-bath @brockfanning spoke to Ann - we won't include this one yet as not very big proportion of indicators affected and more research is needed.
